common/git-hooks/pre-push

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+
+const { execSync } = require('child_process');
+
+function getLocalCommitsNotMerged() {
+  // Fetch updates from the remote repository
+  execSync('git fetch');
+
+  // Get the list of commits not merged to the remote main branch
+  const localCommitsNotMerged = execSync('git log origin/main..HEAD --oneline')
```

Comment:
is "origin" refers to azure/azure-sdk-for-js?  Some developers may choose different convention for that (upstream for example)

Reply:
@jeremymeng True. I have removed the reference to origin. Now, it will compare it to the fork from which the branch was created.

Reply:
Done